torch/autograd/gradcheck.py

```diff
@@ -848,9 +860,9 @@ def to_real_dtype(dtype):
         return dtype
 
 
-def vec_from_tensor(x, generator, downcast_complex=False):
-    # Create a random vector with the same number of elements as x and the same dtype/device
-    # If x is complex, we create a complex tensor with only real component
+def _vec_from_tensor(x, generator, downcast_complex=False):
+    # Create a random vector with the same number of elements as x and the same
+    # dtype/device.
```

Comment:
Why remove the comment about the special case for complex? You still do something special there.

Reply:
Whoops, this should still be there